### PR TITLE
Pirate Translation Updated

### PR DESCRIPTION
**Pull request by hawkblock**
After a year without any edits to this file, hopefully it'll be worth the wait.

### CODE REVIEW

File: cinema/gamemode/localization/pirate.lua [hunk 1]
```diff
@@ -19,21 +19,21 @@ LANG.Set                        = "Set"
...
-LANG.Theater_LostOwnership          = "You have lost your ship due to jumping overboard."
-LANG.Theater_NotifyOwnership        = "You're now the owner of the ship."
-LANG.Theater_OwnerLockedQueue       = "The captain has closed harbor."
-LANG.Theater_LockedQueue            = C(ColHighlight,"%s",ColDefault," has closed the fleet to new ships.")
-LANG.Theater_UnlockedQueue          = C(ColHighlight,"%s",ColDefault," has opened the fleet to new ships.")
+LANG.Theater_RequestFailed          = "The requested ship sank."
+LANG.Theater_Voteskipped            = "The ship has been taken over by rebellious crew!"
+LANG.Theater_ForceSkipped           = C(ColHighlight,"%s",ColDefault," has sank the vessel.")
+LANG.Theater_PlayerReset            = C(ColHighlight,"%s",ColDefault," has swabbed the poopdeck!")
+LANG.Theater_LostOwnership          = "You have lost command of the harbour due to drowning."
```
Comment (zerfgog) [inline cinema/gamemode/localization/pirate.lua]: This file needs quite a bit of `you` -> `ye`. For example, this line should be `Ye've lost command of yer harbour!` or something.

File: cinema/gamemode/localization/pirate.lua [hunk 2]
```diff
@@ -19,21 +19,21 @@ LANG.Set                        = "Set"
...
+LANG.Theater_NotifyOwnership        = "You're now the captain!"
```
Comment (zerfgog) [inline cinema/gamemode/localization/pirate.lua]: `Yer the captain now!`

File: cinema/gamemode/localization/pirate.lua [hunk 3]
```diff
@@ -19,21 +19,21 @@ LANG.Set                        = "Set"
...
+LANG.Theater_OwnerLockedQueue       = "The captain has closed harbour."
+LANG.Theater_LockedQueue            = C(ColHighlight,"%s",ColDefault," has closed the harbour.")
+LANG.Theater_UnlockedQueue          = C(ColHighlight,"%s",ColDefault," has opened the harbour.")
 LANG.Theater_OwnerUseOnly           = "Only the captain can do that."
 LANG.Theater_PublicVideoLength      = "New ship requests are limited to %s second(s) in length."
-LANG.Theater_PlayerVoteSkipped      = C(ColHighlight,"%s",ColDefault," has sunk the SS ",ColHighlight,"(%s/%s)",ColDefault,".")
-LANG.Theater_VideoAddedToQueue      = C(ColHighlight,"%s",ColDefault," has been added to the fleet.")
+LANG.Theater_PlayerVoteSkipped      = C(ColHighlight,"%s",ColDefault," has sunk the ",ColHighlight,"(%s/%s)",ColDefault,"!")
+LANG.Theater_VideoAddedToQueue      = C(ColHighlight,"%s",ColDefault," has sailed into harbour.")
```
Comment (zerfgog) [inline cinema/gamemode/localization/pirate.lua]: these lines should have the "has" removed imo:
"The captain has closed harbour." -> "The captain closed the harbour."
"Blah has closed the harbour." -> "Blah closed the harbour."

File: cinema/gamemode/localization/pirate.lua [hunk 4]
```diff
@@ -88,9 +88,9 @@ LANG.Settings_MuteFocusLabel    = "Silence noises while in quarters"
 LANG.Settings_MuteFocusTooltip  = "Mute theater volume while Garry's Mod is out-of-focus (e.g. you alt-tabbed)."
 
 -- Video Services
-LANG.Service_EmbedDisabled      = "The requested ship is unable to sail."
-LANG.Service_PurchasableContent = "The requested ship is to exspensive to sail."
-LANG.Service_StreamOffline      = "The requested ship is lost."
+LANG.Service_EmbedDisabled      = "The requested ship is unfit for the open sea."
+LANG.Service_PurchasableContent = "The requested ship is too expensive to sail."
```
Comment (zerfgog) [inline cinema/gamemode/localization/pirate.lua]: `is` -> `be` on this and the above line

**Comment by zerfgog** [comment]
I've created a PR in your fork.

**Comment by samuelmaddock** [comment]
I'll merge this PR for now; just submit another if you want to make additional changes.

**Pull request merged**